lib/types/local-date.js

```diff
@@ -78,31 +79,23 @@ function LocalDate(year, month, day) {
    * @type Number
    * @memberof module:types~LocalDate#
```

Comment:
I think that `@memberof` and `name` is not needed anymore, as the prop is defined using `this.propName` below, jsdoc will infer those.